src/test/java/com/nerodesk/om/aws/AwsDocsTest.java

```diff
@@ -49,4 +71,59 @@ public void conformsToEqualsHashCodeContract() {
             .suppress(Warning.TRANSIENT_FIELDS)
             .verify();
     }
+
+    /**
+     * AwsDocs can list all docs.
+     * @throws IOException If something goes wrong.
+     */
+    @Test
+    public void listsDocs() throws IOException {
+        final Bucket bucket = this.mockBucket();
+        final Docs docs = new AwsDocs(bucket);
+        final List<String> expected = Lists.newArrayList(bucket.list(""));
+        MatcherAssert.assertThat(
+            docs.names(),
+            Matchers.equalTo(expected)
+        );
+    }
+
+    /**
+     * AwsDocs can find a doc.
+     * @throws IOException If something goes wrong.
+     */
+    @Test
+    public void findsDoc() throws IOException {
```

Comment:
@pinaf Oh, sorry. I forgot that we have 2 methods. OK for the second test.
And also I wasn't very precise about users docs. What I actually want is the one more test (at leas a puzzle for it) to test that ```AwsDocs. names()``` can return hierarchical names if some files.
You can actually modify the existing test for that.